src/features/live_sensor/index.tsx

```diff
@@ -0,0 +1,109 @@
+import { Car24 } from '@konturio/default-icons';
+import { notificationServiceInstance } from '~core/notificationServiceInstance';
+import { toolbarControlsAtom } from '~core/shared_state';
+import { controlGroup, controlVisualGroup } from '~core/shared_state/toolbarControls';
+import { store } from '~core/store/store';
+import { i18n } from '~core/localization';
+import { sensorDataAtom } from './atoms/sensorData';
+import { sensorResourceAtom } from './atoms/sensorResource';
+import { SENSOR_CONTROL, SENSOR_CONTROL_NAME } from './constants';
+import { collectedPointsAtom } from './atoms/collectedPoints';
+import { hookGeolocation, hookSensors } from './utils';
+import { resourceTriggerAtom, triggerRequestAction } from './atoms/triggerResource';
+import type { Unsubscribe } from '@reatom/core';
+
+export function initSensor() {
+  let accelerometer: Accelerometer;
+  let orientationSensor: AbsoluteOrientationSensor;
+  let geolocation: Geolocation;
+  let gyroscope: Gyroscope;
+  let featureInitializingFailed = false;
+  let resourceUnsubscribe: Unsubscribe;
+
+  try {
+    accelerometer = new Accelerometer();
+    orientationSensor = new AbsoluteOrientationSensor();
+    gyroscope = new Gyroscope();
+    geolocation = navigator.geolocation;
+  } catch (e) {
+    featureInitializingFailed = true;
+  }
+
+  let interval: NodeJS.Timer;
```

Comment:
Is there a place where the "interval" value is set?

Reply:
hehe i forgot to remove it, thanks